modules/ocf_www/manifests/site/www.pp

```diff
@@ -42,6 +49,12 @@
     request_headers     => ['set X-Forwarded-Proto https'],
     proxy_preserve_host => true,
 
+    aliases             => [
+      { alias => '/.well-known/matrix/server',
```

Comment:
should probably put alias on its own line to be consistent with the other sections